bot/code_review_bot/revisions.py

```diff
@@ -133,6 +134,15 @@ def namespaces(self):
             "phabricator.diffphid.{}".format(self.diff_phid),
         ]
 
+    @property
+    def before_after_feature(self):
+        """
+        Randomly run the before/after feature depending on a configured ratio.
+        All the diffs of a revision must be analysed with or without the feature.
+        """
+        random.seed(self.id)
```

Comment:
That does not work, the seed is statically set, but different calls to `random()` will generate different numbers (just try it in a python shell).

You need to remove the random component, and just use the last 4 digits as decimals

Reply:
Just adding a comment here to explain why this was marked as resolved: by calling `random.seed` again, it's like resetting the randomness. So the series of calls `random.seed(X)` + `random.random` always generates the same result.
Please correct me if wrong.

Reply:
BTW, might be worth to add an explanatory comment in the code